painttiming.bs

```diff
@@ -69,9 +102,50 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+A [=generated content pseudo-element=] is a <dfn>paintable pseudo-element</dfn> when all of the following apply:
+* The pseudo-element's [=used=] [=visibility=] is <code>visible</code>.
+* The pseudo-element's [=used=] [=opacity=] is greater than zero.
+* The pseudo-element generates a non-empty [=box=].
+
+A CSS [=image=] |img| is a <dfn>contentful image</dfn> when all of the following apply:
+* |img| is [=url valued=]
+* |img| represents a [=fully decodable image=]
+
```

Comment:
@npm1 @yoavweiss can you take another look at this change before I merge?

Reply:
Definition LGTM.

I want @npm1 to verify that this is not a behavior change from current metric (and if it is, we should probably discuss it some more)

Reply:
I don't think I agree with this wording, and it does seem to be like it's behavior change. Why wait until `fully decodable`? For a progressively rendered image, we want to let FCP be the first time it renders.

Reply:
What would be the correct wording? "loaded" image is a CSS term... And I couldn't find a standard term for an image that is "ready to be presented"

Reply:
If we want to account for partial decodes, then "Partially available" is probably the best definition we can rely on.

Reply:
Actually "available" would work best, as it represents either fully-decodable or partially available.

Reply:
Revised to "available" - WDYT @npm1 @yoavweiss ?

Reply:
WFM

Reply:
WFM2

Reply:
YAY!